### PR TITLE
Bug 1220195 - addons installed from WebIDE appear in settings, r=gasolin

### PR DESCRIPTION
**Pull request by scottwu**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=abf59e00ad44961fe4e536ab7747accf44a15e1d

File: apps/settings/js/modules/addon_manager.js [hunk 1]
```diff
@@ -51,7 +51,16 @@ define(function(require) {
       // manifest, we need to listen to 'downloadsuccess' on the app
       // (not mozApps.mgmt), and that's when manifest will be ready.
       AppsCache.addEventListener('oninstall', (evt) => {
-        evt.application.addEventListener('downloadsuccess', _boundUpdateAddons);
+        var app = evt && evt.application;
+        // Bug 1220195
+        // If installed via WebIDE, _isAddon will return true, and we
```
Comment (yzen) [inline apps/settings/js/modules/addon_manager.js]: nit: can you expand you comment to mention that downloadsuccess never fires for the webide installed addons.

**Comment by yzen** [comment]
Also, would be nice if you could have a test for this, similar to install even:
https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/test/unit/modules/addon_manager_test.js#L303

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=15d3116ef62c6686c44b2fc4eaa44966b452eed8

**Pull request merged**